src/node/rpc/memberfrontend.h

```diff
@@ -361,6 +361,13 @@ namespace ccf
       return false;
     }
 
+    void record_voting_history(
+      Store::Tx& tx, CallerId caller_id, const SignedReq& signed_request)
+    {
+      auto voting_history = tx.get_view(network.voting_history);
+      voting_history->put(caller_id, {signed_request});
```

Comment:
Since this now stores a lot more than votes, it could be renamed? `governance_history` probably, or maybe `member_signatures`?